common/system/tasks/cli/task_cli_sense.cpp

```diff
@@ -122,6 +122,15 @@ void CliTasks::sense() {
     snprintf(cliStr, sizeof(cliStr), " %i", data->pyroContData[i]);
     cli_send(cliStr);
   }
+  snprintf(cliStr, sizeof(cliStr), "\n Pyro Continuity TEST 1-%" PRIu32 ":\n",
+           static_cast<uint32_t>(NUM_PYRO_CONT));
+  cli_send(cliStr);
+  for (int i = 0; i < NUM_PYRO; i++) {
+    snprintf(cliStr, sizeof(cliStr), "Pyro %i : %s \r\n", i,
+             triggerManager_triggerStatusToString(
+                 triggerManager_getExpectedTriggers(i)));
+    cli_send(cliStr);
```

Comment:
Are we talking about pyros, or about triggers?

Reply:
Oh, triggers are the same thing as (pryo | line cutter)?

Reply:
Yes. Right now only Pyros are implemented but I hope to add line cutters later. Also, Should this replace the old output or should I leave both in for now?